screens/Home.js

```diff
@@ -317,6 +318,225 @@ class Home extends React.Component {
   }
 }
 
+Home.defaultProps={
+    mergedBaniData: {
+      "baniOrder": [{
+        "gurmukhi": "gur mMqR",
+        "id": 1,
+        "translit": "gur ma(n)tr"
+      }, {
+        "gurmukhi": "jpujI swihb",
+        "id": 2,
+        "translit": "japujee saahib"
+      }, {
+        "gurmukhi": "Sbd hzwry",
+        "id": 3,
```

Comment:
Veerji is it necessary to hard code this data here? If you will make this prop required it can solve this problem. Also, I don't think it's ideal to hard code the data that's eventually going to be fetched from the database. Just in case we make any major change in DB which changes the placement and id of Banis this code will break.

Reply:
Well I made this required but got error for eslint. Should I turn it off to check propTypes?

Reply:
No @amitojsingh veerji, when there's no data in that prop just write it as an empty object. The exact data should be fetched from the database.

Reply:
sure I made this empty

Reply:
Did it work that way?

Reply:
I have verified it on my end it's working smoothly :slightly_smiling_face: